src/fswAlgorithms/effectorInterfaces/torqueScheduler/torqueScheduler.c

```diff
@@ -22,6 +22,8 @@
 #include "torqueScheduler.h"
 #include "string.h"
 
+/* Support files */
+#include "architecture/utilities/macroDefinitions.h"
```

Comment:
:small_red_triangle_down: This include is unused

Reply:
Removed it.